### PR TITLE
Add integration with project explorer

### PR DESCRIPTION
**Pull request by Krzysztof-Cieslak**
screenshot from 2018-03-13 14-37-05

### CODE REVIEW

File: src/Mechanic.fs [hunk 1]
```diff
@@ -57,8 +61,10 @@ let activate (context : Vscode.ExtensionContext) =
                 state.Add project
         ) |> ignore
 
-        Vscode.commands.registerCommand("mechanic.run", fun _ ->
-            Promise.start runInScope
+        Vscode.commands.registerCommand("mechanic.run", fun input ->
+            let m = unbox<ProjectExplorerModel option> input
+            printfn "M: %A" m
```
Comment (MangelMaxime) [inline src/Mechanic.fs:66]: We probably should remove this log

Comment (Krzysztof-Cieslak) [reply src/Mechanic.fs:66]: Woops

Comment (forki) [reply src/Mechanic.fs:66]: will do

Comment (Krzysztof-Cieslak) [reply src/Mechanic.fs:66]: Already done ;P

Comment (forki) [reply src/Mechanic.fs:66]: awesome. and release process works! https://marketplace.visualstudio.com/items?itemName=Ionide.mechanic

**Pull request merged**